### PR TITLE
Introduce api_manage subscription_manage scopes

### PR DESCRIPTION
**Pull request by chathuranga95**
Fixes: https://github.com/wso2-enterprise/choreo/issues/6722

This PR introduces the following scopes for REST apis.

**apim:api_manage** is capable of performing the operations under `apim:api_view, apim:api_create, apim:api_publish, apim:api_delete, api_import_export, apim:api_product_import_export, api_product_import_export, api_generate_key`

**apim:subscription_manage** is capable of performing the operations under `apim:subscription_view,  apim:subscription_block`

**apim:app_manage** is capable of performing the operations under `apim:app_manage, apim:app_import_export`

### CODE REVIEW

**Review by sumedhassk — APPROVED** [review]
LGTM

File: components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/resources/publisher-api.yaml [hunk 1]
```diff
@@ -4658,6 +4735,7 @@ paths:
       security:
         - OAuth2Security:
             - apim:api_view
+            - apim:api_manage
```
Comment (malinthaprasan) [inline components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/resources/publisher-api.yaml:4738]: It is better to use a new group for this as this is a seperate API.
```suggestion
            - apim:throttling_policy_manage
```

File: components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/resources/publisher-api.yaml [hunk 2]
```diff
@@ -4708,6 +4786,7 @@ paths:
...
```
Comment (malinthaprasan) [inline components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/resources/publisher-api.yaml:4789]: Lets do the same for all the throttling policy related APIs

File: components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/resources/publisher-api.yaml [hunk 3]
```diff
@@ -7233,6 +7361,7 @@ paths:
...
```
Comment (malinthaprasan) [inline components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/resources/publisher-api.yaml:7364]:
```suggestion
            - apim:shared_scope_manage
```

File: components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/resources/publisher-api.yaml [hunk 4]
```diff
@@ -7309,6 +7438,7 @@ paths:
...
```
Comment (malinthaprasan) [inline components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/resources/publisher-api.yaml:7441]:
```suggestion
            - apim:shared_scope_manage
```

File: components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/resources/publisher-api.yaml [hunk 5]
```diff
@@ -7400,6 +7530,7 @@ paths:
       security:
         - OAuth2Security:
             - apim:api_create
+            - apim:api_manage
```
Comment (malinthaprasan) [inline components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1/src/main/resources/publisher-api.yaml:7533]:
```suggestion
            - apim:api_manage
            - apim:shared_scope_manage
```

**Comment by malinthaprasan** [comment]
Suggested Publisher REST API Scope Hierarchy:

```
apim:publisher_settings

apim:api_manage # new
    apim:api_view
    apim:api_create
    apim:api_delete
    apim:api_publish
    apim:api_import_export
    apim:document_manage
        apim:document_create # deprecate
    apim:api_mediation_policy_manage # new
    apim:client_certificates_manage # new
        apim:client_certificates_view
        apim:client_certificates_add
        apim:client_certificates_update
    apim:ep_certificates_manage # new
        apim:ep_certificates_view
        apim:ep_certificates_add
        apim:ep_certificates_update
        
apim:comment_manage  # new
    apim:comment_view
    apim:comment_write

apim:subscription_manage
    apim:subscription_view
    apim:subscription_block
    
apim:tier_manage # new from admin api
    apim:tier_view # new from admin api

apim:mediation_policy_manage
    apim:mediation_policy_view
    apim:mediation_policy_create
    apim:api_mediation_policy_manage

apim:shared_scope_manage

apim:api_product_import_export
```

**Review by malinthaprasan — APPROVED** [review]

**Pull request merged**